migrations/versions/e1b6d52fd6ea_.py

```diff
@@ -0,0 +1,22 @@
+from alembic import op
+import sqlalchemy as sa
+
+revision = 'e1b6d52fd6ea'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        'profile',
+        sa.Column('id', sa.String(length=8), nullable=False),
+        sa.Column('name', sa.String(length=128), nullable=False),
```

Comment:
I assume if we update this name, we will keep the events in another table and this will be the latest `name`

Reply:
I'll just read through http://www.kalzumeus.com/2010/06/17/falsehoods-programmers-believe-about-names/ for pitfalls

Reply:
Is that 128 bytes or 128 Unicode chars? I guess the first

Reply:
I imagine so. Wonder what ORCID's length is. We might want to remove this for now, as they have first name, last name and published name.

Reply:
If we keep MySQL we should make sure that it's using `utf8mb4`.

Reply:
> If we keep MySQL we should make sure that it's using utf8mb4
I wasn't going to mention any of the many reasons why we should avoid mysql in favour of postgresql, but it's ridiculous encoding problems would be near the top of my list.

Reply:
In Postgres (and MySQL 5) it's chars not bytes.

Guess https://github.com/ORCID/ORCID-Source/blob/a78c2ad843943ab0a4060a536118d4b8f4e7ab1e/orcid-persistence/src/main/resources/schema.sql#L476-L477 means that the ORCID name could be 301 chars.

Reply:
curiously, these limits are hardcoded in the schema
https://github.com/ORCID/ORCID-Source/blob/master/orcid-model/src/main/resources/record_2.0/personal-details-2.0.xsd#L110-L133
along with the interesting
https://github.com/ORCID/ORCID-Source/blob/master/orcid-model/src/main/resources/common_2.0/common-2.0.xsd#L1440-L1457

Reply:
Huh. Just seen that the DB length looks to have been removed - https://github.com/ORCID/ORCID-Source/blob/5f5508183ef971d2d7529af7d6a84554aa860749/orcid-persistence/src/main/resources/db/updates/varchar_to_text.xml#L152-L156

Reply:
this is not a field that should be indexed, so using an arbitrary `text` shouldn't have a big impact?

Reply:
We might want searching by name in the future (I'm envisaging someone using Journal CMS), but quickly reading http://rachbelaid.com/postgres-full-text-search-is-good-enough/ doesn't look like it'd be a problem. So let's change it.